docs/src/howto.md

```diff
@@ -121,3 +121,96 @@ Action(1:100, (args...)->rect(pos(:my_circle)+Point(-10, -100), 20, 20, :fill))
 
 In this animation the position of the circle is saved inside `:my_circle` and can be used with `pos(:my_circle)` inside the `rect` function.
 
+## How can I show a text being drawn?
+
+A `text` can appear as every other object with `appear(:fade)` and `appear(:scale)` but it also has a special [`appear`](@ref) functionality called 
+`:draw_text`.
```

Comment:
Maybe: 

"A `text` can appear as _any_ other object with `appear(:fade)` and `appear(:scale)`. However, it also has a special [`appear`](@ref) functionality called `:draw_text`."